src/components/tasks/card/index.tsx

```diff
@@ -79,22 +79,23 @@ const Card: FC<Props> = ({
     const daysLeft = endDate.diff(new Date(), 'days')
 
     // It provides the percentage of days left
-    const percentageofDays = daysLeft/totalDays * 100
-    return percentageofDays
+    const percentageofDaysLeft = daysLeft/totalDays * 100
+    return percentageofDaysLeft
   }
   
   function handleProgressColor(percentCompleted: number, startedOn: string, endsOn: string): string {
-    const percentageofDays = getPercentageOfDays(startedOn, endsOn)
-    if(percentageofDays >= percentCompleted) {
+    const percentageofDaysLeft = getPercentageOfDays(startedOn, endsOn)
```

Comment:
Should we rename the function too?

Reply:
How about refactoring it to setProgressBarColor?

Reply:
1. I think he meant `getPercentageOfDays` could be renamed to `getPercentageOfDaysLeft`. Not sure though.
2. `handleProgressColor` looks fine.

Reply:
Got it. Will refactor getPercentageOfDays to getPercentageOfDaysLeft.